tests/dragonfly/acl_family_test.py

```diff
@@ -332,8 +336,16 @@ async def test_good_acl_file(df_factory, tmp_dir):
     await client.execute_command("ACL LOAD")
     result = await client.execute_command("ACL list")
     assert 2 == len(result)
-    assert "user MrFoo on #ea71c25a7a60224 -@all" in result
+    assert (
+        "user MrFoo on #ea71c25a7a60224 #a6864eb339b0e1f -@all" in result
+        or "user MrFoo on #a6864eb339b0e1f #ea71c25a7a60224 -@all"
```

Comment:
is "in result" missing for the second part of the or?

Reply:
@adiholden I think it works that way as well because the test failed previously and it passed now so maybe it's some specific syntax with python. I fixed it to be on the safe side